recipe/meta.yaml

```diff
@@ -25,6 +29,11 @@ requirements:
 
 test:
   commands:
+    - test -f "${PREFIX}/share/zoneinfo/leapseconds"
+    - test -f "${PREFIX}/share/zoneinfo/iso3166.tab"
+    - test -f "${PREFIX}/share/zoneinfo/zone1970.tab"
+    - test -f "${PREFIX}/share/zoneinfo/zone.tab"
+    - test -f "${PREFIX}/share/zoneinfo/tzdata.zi"
```

Comment:
These files are included in the Python package. Which means this initial build will carry the same set of files (albeit under a different path) as https://pypi.org/project/tzdata/2020.1/ .

I don't know if their inclusion in the Python package was intentional or not. (And I have no experience in dealing with tzdata, so can't tell when they are needed.)
I'll open an issue for that at their issue tracker.

Reply:
Which of these do you think shouldn't be installed?

`tzdata.zi` definitely should be, the tab files and leap seconds files were included intentionally as they are part of the installed distribution (we take whatever the upstream project installs, see [this](https://github.com/python/tzdata/blob/842c5384d0a02c30de513b33277369d488f374c2/update.py#L86-L99)).

Reply:
Oh, maybe it makes sense to include them all. Unfortunately I have no in-depth knowledge of this at all.
Hence I included all files you have in https://github.com/python/tzdata in this one as well.

Reply:
Thanks for reviewing this an confirming that it was indeed intentional!